test/branchTest.js

```diff
@@ -20,37 +25,82 @@ describe("Branch", () => {
 
   describe("getSlot", async () => {
     it("Returns the uint16 in the correct position", async () => {
-      const result = await branchInstance.getSlot(node, 3)
-      expect(result).to.be.equal(0x33333333)
+      const expectations = [
+        0x00000000, 0x11111111, 0x22222222, 0x33333333, 0x44444444, 0x55555555,
+        0x66666666, 0x77777777,
+      ]
+      for (let slot = 0; slot < 8; slot++) {
+        const result = await branchInstance.getSlot(node, slot)
+        expect(result).to.equal(expectations[slot])
```

Comment:
Worth adding an error message that will indicate the problematic slot:
```suggestion
        expect(result).to.equal(expectations[slot], `unexpected result at position ${slot}`)
```

Reply:
[b665a29](https://github.com/keep-network/sortition-pools/pull/172/commits/b665a29858ed4d95052032a36ea08f143676740a)